woocommerce/woocommerce-ios
# Orders - empty view pull to refresh

Part of #6356


### Description
This PR adds pull to refresh to the empty view controller in `OrderListViewController`.
- Remove empty config from init to send the pull to refresh callback.
- Add a pull to refresh callback to empty view when there are no orders available in store.

### Testing instructions


- Login to the app with a valid account
- Turn off the internet connection
- Switch to the Orders tab. Without internet connection, no data can be fetched, so empty views will be presented shortly afterward.
- Turn on the internet connection.
- Pull down to refresh and observe that orders are loaded now.

### Screenshots


https://user-images.githubusercontent.com/524475/171361287-0f46d6af-c989-485a-901c-cabd4790019e.mov



---
